src/py42/modules/securitydata.py

```diff
@@ -107,14 +108,14 @@ def get_all_user_security_events(
         min_timestamp=None,
         max_timestamp=None,
     ):
-        """Gets legacy security events for the user with the given UID.
+        """Gets file activity events for the user with the given UID.
```

Comment:
As an end user, how do I know that this is FED and not AED? That is why the word "legacy" was there. I'm curious as to the feedback that prompted it's removal and/or alternative ideas for how we can convey the limitations of this method.

Reply:
Does `file activity` say enough? Cloud activity is still file activity.

Reply:
@annie-payseur thoughts?

Reply:
End users don't know the terms FED and AED at all - those are internal terms/acronyms. "Legacy security events" doesn't tell them much, either. To your point on file activity & cloud activity, we could say "endpoint file activity" here?

Reply:
The header of the checkboxes for enabling FED in Console is "Endpoint Monitoring" so I think that's a good compromise.

Reply:
`legacy Endpoint Monitoring` works for me, I still want to indicate that this is the "old" way of doing things.